utils/singularity/def/capice-3.0.0rc2.def

```diff
@@ -0,0 +1,16 @@
+Bootstrap: library
+From: ubuntu:20.04
+
+%post
+    apt-get -y update
+
+    apt-get -y install software-properties-common
+    add-apt-repository universe
+    apt-get -y install python3 python3-pip python3-wheel
+
+    python3 -m pip install capice==3.0.0rc2
+
```

Comment:
Clean up to reduce final image size (alternatively: use build/final stage to realize this)